projects/media-viewer/src/lib/toolbar/right-pane/right-pane.component.html

```diff
@@ -9,28 +15,31 @@
           data-l10n-id="open_file">
     <span data-l10n-id="open_file_label">Open</span>
   </button>
-  <button *ngIf="enableRedactions"
+  <div class="toolbarButtonSpacer"></div>
+  <button *ngIf="enableRedaction && !icpSimplifiedToolbar"
```

Comment:
these two checks seem to be doing the same thing, so I think we should be using one of them:
- !(toolbarEvents.icp.enabled | async)
- !icpSimplifiedToolbar